docs/view-style-props.md

```diff
@@ -326,4 +326,4 @@ Controls whether the `View` can be the target of touch events.
 
 | Type                                          |
 | --------------------------------------------- |
-| enum('auto', 'box-none', 'box-only', 'none' ) |
+| enum(`'auto'`, `'box-none'`, `'box-only'`, `'none'` ) |
```

Comment:
All other instances of `enum` values in this document appear enclosed in these back-ticks:

<img width="835" alt="Screenshot 2024-09-29 at 21 03 57" src="https://github.com/user-attachments/assets/36023cb6-7a72-4eb6-849f-c2f1ea10dfca">
<img width="830" alt="Screenshot 2024-09-29 at 21 04 06" src="https://github.com/user-attachments/assets/8e32da2c-6c81-4ce0-8f18-8153ff600685">
<img width="832" alt="Screenshot 2024-09-29 at 21 04 13" src="https://github.com/user-attachments/assets/77338b84-d2fe-49dc-b58e-35a5d25bb82b">